deploy/determined_deploy/aws/templates/simple.yaml

```diff
@@ -95,6 +95,11 @@ Parameters:
     Description: Whether to allow CORS requests or not
     Default: false
 
+  EnableAutoTLS:
```

Comment:
Question: why are we not changing all of the templates to support this?  Is this really only useful in testing?

Reply:
What I'm getting at is, are we adding a large amount of test-only-code to something that we give to users as a means of deploying the cluster?  Or would users find this feature to be useful?

Reply:
I think this would be useful for users, yeah.

Reply:
Seems like we should just ship this as a feature then.  Also, if the cost of duplicating this work across all the templates is too high to ship all of them at once, my sense is that we should be taking a different approach (like embedding the autotls script in the agent or something).

Reply:
Okay, I can go ahead and update all the templates. I'm open to other approaches, but not sure what you mean by that one -- did you mean "master" rather than "agent"? The master could generate a cert pretty nicely (in Go rather than the shell, even), but going down that route is going to involve some more design decisions that I'd have to spend more time on.

Reply:
Well what I was thinking was in the agent ami, but yeah, that doesn't make sense.

What would it look like to do it in the master? I feel like copy/pasting the file-generating-script that is currently there 5 times sounds... undesirable.  Could you just have an auto-tls feature on the master that autogenerated certs or something?

Also... how does the webui work with this change as it is written right now?